docker/test/docker-compose.yml

```diff
@@ -118,24 +119,24 @@ services:
       orbs-network:
         ipv4_address: 192.168.199.5
 
-  orbs-e2e:
-    image: orbs:build
-    environment:
-      API_ENDPOINT: http://orbs-network-node-3:8080/api/v1/
-      STRESS_TEST_NUMBER_OF_TRANSACTIONS: 5000
-      STRESS_TEST_FAILURE_RATE: 20
-      STRESS_TEST_TARGET_TPS: 100
-      STRESS_TEST: 'true'
-      ETHEREUM_ENDPOINT: http://ganache:8545/
-      ETHEREUM_PRIVATE_KEY: f2ce3a9eddde6e5d996f6fe7c1882960b0e8ee8d799e0ef608276b8de4dc7f19 # this keypair is generated from the mnemonic passed to ganache on startup
-      ETHEREUM_PUBLIC_KEY: 037a809cc481303d337c1c83d1ba3a2222c7b1b820ac75e3c6f8dc63fa0ed79b18
-      EXTERNAL_TEST: 'true'
-    command: sh -c "sleep 15 && ./test.adapters.sh"
-    sysctls:
-      net.core.somaxconn: 25000
-    networks:
-      orbs-network:
-        ipv4_address: 192.168.199.7
+  # orbs-e2e:
```

Comment:
why comment out? delete!

Reply:
You are right - will be deleted.  @electricmonk